docs/docs/sample-scripts/spo/add-site-collection-admin-using-csv-file.md

```diff
@@ -0,0 +1,64 @@
+# Add a Site Collection Admin using a csv file
+
+## Word of caution
```

Comment:
We could turn this paragraph into an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to make it more visible